src/main/java/org/folio/notes/controller/NoteTypesController.java

```diff
@@ -1,37 +1,48 @@
 package org.folio.notes.controller;
 
+import java.net.URI;
 import java.util.UUID;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
 
 import org.folio.notes.domain.dto.NoteType;
 import org.folio.notes.domain.dto.NoteTypeCollection;
 import org.folio.notes.rest.resource.NoteTypesApi;
+import org.folio.notes.service.NoteTypesService;
 
+@RestController
+@RequiredArgsConstructor
 public class NoteTypesController implements NoteTypesApi {
 
+  private final NoteTypesService noteTypesService;
+
   @Override
   public ResponseEntity<NoteType> createNoteType(NoteType noteType) {
-    return NoteTypesApi.super.createNoteType(noteType);
+    NoteType newType = noteTypesService.createNoteType(noteType);
+    return ResponseEntity.created(URI.create("/note-types/" + newType.getId())).body(newType);
   }
 
   @Override
   public ResponseEntity<Void> deleteNoteType(UUID id) {
-    return NoteTypesApi.super.deleteNoteType(id);
+    noteTypesService.removeNoteTypeById(id);
+    return ResponseEntity.noContent().build();
   }
 
   @Override
   public ResponseEntity<NoteType> getNoteType(UUID id) {
-    return NoteTypesApi.super.getNoteType(id);
+    return ResponseEntity.ok(noteTypesService.getById(id));
```

Comment:
Because you have `updateNoteType`
```suggestion
    return ResponseEntity.ok(noteTypesService.getNoteType(id));
```